gwcli/storage.py

```diff
@@ -402,7 +402,7 @@ def ui_command_info(self, image_id):
         else:
             self.logger.error("disk name provided does not exist")
 
-    def ui_command_delete(self, image_id):
+    def ui_command_delete(self, image_id, delete_rbd=None):
```

Comment:
Nit: since you would need to pass the optional to keep the image, perhaps it should renamed to something like ```keep_image``` or ```preserve_image```? Also, can you add it to the help text below?